File: java/EffectRelatedITTs.java
import javax.ws.rs.client.Client;
import javax.ws.rs.client.ClientBuilder;
import javax.ws.rs.client.WebTarget;
import javax.ws.rs.core.MediaType;
import javax.ws.rs.core.MultivaluedHashMap;
import javax.ws.rs.core.MultivaluedMap;

public class EffectRelatedITTs {

    private static final String API_BASE_URL = "https://www.surfwatchanalytics.com:443/v2";

    public static void main( String[] args ) throws Exception {
        
        String effectId = "18786";
        Client restClient = ClientBuilder.newClient();  
        WebTarget target = restClient.target( API_BASE_URL )
                .path( "/cyberTags/currentAnalytics/effectITTs/"+ effectId )
                .queryParam( "yesterday", "true" );

        MultivaluedMap<String, Object> headers = new MultivaluedHashMap<>();
        headers.add( "app_id", "your_app_id" );
        headers.add( "app_key", "your_app_key" );
        
        String response = target
                .request( MediaType.APPLICATION_JSON )  // alternately set "Content-Type" header
                .headers( headers )
                .get( String.class );
        
        System.out.print( response );
    }    
}

File: ruby/effect_related_itts.rb
require 'rest_client'
require 'json'

effect_id = '18786' # Compromised website
url = "https://www.surfwatchanalytics.com:443/v2/cyberTags/currentAnalytics/effectITTs/#{ effect_id }"

# Tags about compromised websites from yesterday
header = {
  'content_type' => 'application/json',
  'app_key' => ENV['SURFWATCH_ANALYTICS_APP_KEY'],
  'app_id' => ENV['SURFWATCH_ANALYTICS_APP_ID'],
  params: {
    'yesterday' => true
  }
}
response = RestClient.get(url, header)
results = JSON.parse(response)